Rakefile

```diff
@@ -6,24 +6,27 @@ require 'rubycritic_small_badge'
 require 'rubycritic/rake_task'
 require 'sandi_meter/file_scanner'
 
-RubyCriticSmallBadge.configure do |config|
-  config.minimum_score = ENV.fetch('RUBYCRITICLIMIT', 90.0)
-  config.output_path = ENV.fetch('RUBYCRITPATH', 'badges/app')
-end
+{ spec: 65, app: 90 }.each do |dir, percentage|
+  RubyCriticSmallBadge.configure do |config|
+    config.minimum_score = percentage
+  end
```

Comment:
I loved that! In fact we need something similar...

Reply:
I'm kind of confused, where does this config come from?

Reply:
it's only to not duplicate code in configuration of rubycritic in directory lib and app